website/docs/topics/feature-flags/best-practices-using-feature-flags-at-scale.md

```diff
@@ -0,0 +1,350 @@
+  - [13. Never reuse feature flag names.](#13-never-reuse-feature-flag-names)
+  - [14. Avoid giant feature flag targeting lists.](#14-avoid-giant-feature-flag-targeting-lists)
+  - [15. Do not define business logic using feature flags.](#15-do-not-define-business-logic-using-feature-flags)
+- [Staying sane while managing feature flags at scale](#staying-sane-while-managing-feature-flags-at-scale)
+
+## Organizing your team and internal processes to support feature flags
+
+Before you add your first feature flag, you need to think about how organizational structure and processes that live outside of code will affect your feature flag deployments.
+
+### 1. Align feature flags with organizational and application design.
+
+> Any organization that designs a system will produce a design whose structure is a copy of the organization's communication structure.
+
+– Melvin Conway
+
+![feature-flag-orginization-methods](https://github.com/ferrantim/unleash/assets/3068350/7aa67c51-f085-4367-849b-bb6e06a3e754)
+
+There is no getting around [Conway’s law](https://martinfowler.com/bliki/ConwaysLaw.html). Applications tend to resemble the organizations that create them, and your feature flag system is no exception. Rather than fighting this law, acknowledge and embrace it, and pick a way to organize your feature flags that reflects your organization.
+
+A typical Fortune 500 company typically has thousands of software developers.  Organizing your feature flag system based on “applications” when your company is organized based on business units or products will lead to a poor experience and additional complexities in managing flag permissions.
+
+Users should be shown information about any flags related to their work, whether on a team, an application, a product, or a project basis. This includes inspecting the flag’s configuration and tracking updates to the flag.
+
+Achieve this by grouping related flags together based on the most appropriate level of abstraction. If you work on static teams that see a feature through from start to finish, group flags by team. If you form project-based teams, group flags by project.
+
+Taken to the extreme, you might be tempted to put all flags into one giant group. Avoid that temptation because it will create permission complexities and information overload.
+
+In an organization that actively uses feature flags, there will invariably be a large number of flags and an even larger number of changes made to those flags. This can quickly become overwhelming and a user may end up losing track of updates that are important to them. 
+
+Removing information and updates you don’t care about makes the things you care about stand out that much more. 
+
+Example: Do all members of Team A need updates on Project X, even if only one member of Team A is working on it? If so, include all team members in Project X’s group. Otherwise, only include the member who works on project X. 
```

Comment:
```suggestion
**Example**: Do all members of Team A need updates on Project X, even if only one member of Team A is working on it? If so, include all team members in Project X’s group. Otherwise, only include the member who works on project X. 
```